modules/@apostrophecms/modal/ui/apos/mixins/AposDocsManagerMixin.js

```diff
@@ -39,24 +47,20 @@ export default {
     toggleRowCheck(id) {
       if (this.checked.includes(id)) {
         this.checked = this.checked.filter(item => item !== id);
-        if (this.checkboxes[id]) {
-          this.checkboxes[id].value.data = this.checkboxes[id] && [];
```

Comment:
The `value.data` isn't actually used in the checkboxes. Plus we're not using the `checkboxes` object anymore.